packages/server/src/fhir/operations/export.ts

```diff
@@ -68,6 +75,12 @@ async function exportResourceType(exporter: BulkExporter, project: Project, reso
     if (!linkNext?.url) {
       break;
     }
+    const params = new URLSearchParams(linkNext.url);
```

Comment:
We should not make assumptions about `linkNext.url` due to potential changes in that URL format.  See:
1. https://github.com/medplum/medplum/issues/1281
2. https://github.com/medplum/medplum/discussions/1231#discussioncomment-4386350 for longer discussion

Instead, you might as well just use `offset += maxResources;`